components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/statusview/StatusView.kt

```diff
@@ -0,0 +1,141 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import uk.gov.hmrc.components.compose.atom.button.PrimaryButton
+import uk.gov.hmrc.components.compose.atom.button.SecondaryButton
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+
+object StatusView {
+
+    const val NO_ICON: Int = -1
+
+    enum class BodyAlignment(val alignment: TextAlign) {
+        CENTER(TextAlign.Center),
+        START(TextAlign.Start),
+    }
+
+    @Composable
+    operator fun invoke(
+        modifier: Modifier = Modifier,
+        title: String,
+        body: String = "",
+        bodyContentDesc: String = "",
+        bodyAlignment: BodyAlignment = BodyAlignment.CENTER,
+        textColor: Color = HmrcTheme.colors.hmrcBlack,
+        icon: Int,
+        iconTint: Color = HmrcTheme.colors.hmrcBlack,
+        primaryButtonText: String = "",
+        secondaryButtonText: String = "",
+        infoText: String = "",
+        onPrimaryButtonClickedListener: () -> Unit = {},
+        onSecondaryButtonClickedListener: () -> Unit = {}
+    ) {
+        Column(
+            modifier = modifier,
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+
+            //region Icon
+            if (icon != NO_ICON) {
+                Image(
+                    painter = painterResource(id = icon),
+                    contentDescription = "",
+                    modifier = Modifier
+                        .height(HmrcTheme.dimensions.hmrcIconSize100)
+                        .width(HmrcTheme.dimensions.hmrcIconSize100),
+                    colorFilter = ColorFilter.tint(iconTint),
+                    contentScale = ContentScale.Fit,
+                )
+                Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing24))
+            }
+            //endregion
+
+            //region Title
+            Text(
+                text = title,
+                style = HmrcTheme.typography.h5.copy(color = textColor),
+                modifier = Modifier.fillMaxSize(),
+                textAlign = TextAlign.Center
+            )
```

Comment:
Would it work by using `Heading5` here?

Reply:
Yes we can, but one problem I am facing Heading5 has no style parameter to change text colour.

Reply:
One thing I think we lose here by not using the Heading component is the talkback announcement.
Can we use the base Heading component and provide the style directly instead? 

```
Heading(text = text, style = typography.h5.copy(color = textColor), modifier = modifier)
```

Is there another way we can achieve the text alignment while using this component still? If this isn't possible, we should just add the semantics here instead.
```
modifier = modifier.semantics { heading() }
```